src/SamplesApp/SamplesApp.UITests/UnoSamples_Tests.Keyboard.cs

```diff
@@ -39,76 +39,100 @@ public void Keyboard_Textbox_InsideScrollViewer_Validation()
 			Assert.AreEqual(string.Empty, multilineTextBox.GetDependencyPropertyValue("Text")?.ToString());
 			Assert.AreEqual(string.Empty, numberTextBox.GetDependencyPropertyValue("Text")?.ToString());
 
+			var initialScreenshot = TakeScreenshot("Initial state");
 			{
 				// Setting focus on normalTextBox
 				_app.FastTap(normalTextBox);
 				_app.Wait(1);
-				TakeScreenshot("0 - Focus on normalTextBox ", ignoreInSnapshotCompare: true);
+				//TakeScreenshot("0 - Focus on normalTextBox ", ignoreInSnapshotCompare: true);
+				var screenshot1 = TakeScreenshot("0 - Focus on normalTextBox ", ignoreInSnapshotCompare: true);
+				ImageAssert.AreNotEqual(initialScreenshot, screenshot1);
 
 				// Removing focus on normalTextBox
 				_app.TapCoordinates(0f, 0f);
+				_app.DismissKeyboard();
 				_app.Wait(1);
-				TakeScreenshot("0 - Remove Focus on normalTextBox", ignoreInSnapshotCompare: AppInitializer.GetLocalPlatform() == Platform.Android /*Keyboard predicted text can change*/);
+				//TakeScreenshot("0 - Remove Focus on normalTextBox", ignoreInSnapshotCompare: AppInitializer.GetLocalPlatform() == Platform.Android /*Keyboard predicted text can change*/);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4978073)

Reply:
i just want a final list of snapshot errors. After that i will add more in this. that's why i told you to ignore this PR for now. I am just investigation the cause and solution for error.